src/komponenter/kalkulator/kalkulator.tsx

```diff
@@ -38,6 +39,32 @@ const Kalkulator = () => {
     const [sisteDagI18mndsPeriode, setSisteDagI18mndsPeriode] = useState<Dayjs>(
         dagensDato
     );
+    const [tidslinjeObjekter, setTidslinjeObjekter] = useState<
```

Comment:
Jeg foreslår å kalle denne for `tidslinje` i stedet for `tidslinjeObjekter`, siden det er det vi har brukt andre steder

Reply:
Hvorfor ikke gjøre dette i `Tidslinje`, egentlig? Så slipper `Kalkulator` å håndtere logikk som er spesifikk for tidslinjen

Reply:
nå brukes tidslinja også i utregningskolonne, så det gjøres en mindre konstruksjon av den. I Tillegg tror jeg det må være sånn for at tidslinjehtml-objektene skal kunne få inn sin egen størrelse i prosent. Har ihvertfall ikke funnet en work-around på det.